### PR TITLE
Refactor order success page

### PR DESCRIPTION
**Pull request by brettski**
## v5.0.6

- refactor order success page to improve message and add latest THAT offerings
- Remove bing update ping from `release-success` as it's no longer supported by bing
- fixed shared profile phone number field formatting
  - when phone number `null` from datastore error calling `event.target.value.replace()` on line 39
  - API will only accept a valid phone number or a null value, updated so an empty phone number is a null value. Adjusted Zod accordingly.
  - Updated Zod rules to allow RegEx approved phone number or null. Optional doesn't work in the case with a regex. It is recommended to use `z.union()` or `.or()` in this case.

### Testing

- success page on ticket purchase: `/orders/success?EL=tx&M=0&B=0&TL=at`
- success page on ticket purchase w/ bulk: `/orders/success?EL=tx&M=0&B=4&TL=at`
- success page on membership purchase: `/orders/success?EL=tx&M=1&B=0` or `/orders/success?EL=thatus&M=1&B=0`

- Test shared phone number:
  - no field from db
  - null field from db
  - USA, India and German number formats
  - Clearing and saving value

### CODE REVIEW

File: src/routes/(root)/orders/success/+layout.svelte [hunk 1]
```diff
@@ -12,5 +12,7 @@
 
 <slot />
 <div class="mt-24">
+	<!-- svelte-ignore a11y-missing-content -->
+	<a name="newsletter"></a>
```
Comment (csell5) [inline src/routes/(root)/orders/success/+layout.svelte:16]: why do we have an empty link?

File: src/lib/formSchemas/sharedProfile.js [hunk 2]
```diff
@@ -14,5 +14,6 @@ export default z.object({
 	company: z.string().trim().nullable().optional(),
 	jobTitle: z.string().trim().nullable().optional(),
 
-	phone: z.string().trim().regex(PHONE_NUMBER_REGEX).optional() // todo.. this feels werid.
+	// THAT api cannot accept '' as phone value. Must be valid phone # or null
+	phone: z.string().trim().regex(PHONE_NUMBER_REGEX).or(z.literal(null)) // todo.. this feels werid.
```
Comment (csell5) [inline src/lib/formSchemas/sharedProfile.js:18]: why not just .nullable() rather than the or

**Review by csell5 — CHANGES_REQUESTED** [review]

File: src/routes/(root)/orders/success/+layout.svelte (hunk 1 above)
Comment (brettski) [reply src/routes/(root)/orders/success/+layout.svelte:16]: I added this as an in-page anchor to which is referenced higher in the success page to scroll down to the sign-up. The empty "sign-up for the newsletter" bullet had no action to it.

File: src/lib/formSchemas/sharedProfile.js (hunk 2 above)
Comment (brettski) [reply src/lib/formSchemas/sharedProfile.js]: Updated per suggestion and retested

**Comment by csell5** [comment]
in the future let's stop batching items in prs. The Bing fix could have been its own as well as the phone number bug. It makes it harder to reference later when it's buried in a general pr.

**Review by csell5 — APPROVED** [review]

**Pull request merged**